types.go

```diff
@@ -115,6 +115,7 @@ const (
 	_TaskFDQuery
 	_MapLookupAndDeleteElem
 	_MapFreeze
+	_BTFGetNextID
```

Comment:
At the moment, the BTF part is no implemented. Calling `objGetNextID()` from `syscalls.go` within `internal/btf/*.go` would introduce an import cycle.

Reply:
I didn't move the function `objGetNextID` into `internal/` as I wanted to avoid making it public.